backend/ModelView/SessionPlayer.cs

```diff
@@ -5,9 +5,8 @@ namespace IP5.Model
 {
 	public class SessionPlayer
 	{
-		public Guid SessionId { get; set; }
-		public Guid PlayerId { get; set; }
-
+		public string SessionCode { get; set; }
+		public string PlayerCode { get; set; }
 		public Session Session { get; set; }
 		public Player Player { get; set; }
```

Comment:
Isn't the `PlayerCode` the only member used?
Is `SessionCode`, `Session` and `Player` members required / used?

Reply:
On put Im using both (PlayerCode & SessionCode) to create the SessionPlayer record, but on get I'm fetching only the Player atm